npc/quests/skills/merchant_skills.txt

```diff
@@ -41,7 +41,7 @@ alberta,83,96,5	script	Necko	98,7,7,{
 			mes "Ppyakikakikakakakakakakaka!!";
 			close;
 		}
-		else if (JobLevel < 15) {
+		else if (JobLevel < 15 && BaseClass = Job_Merchant ) {
```

Comment:
Like I wrote before this dialogue should appear for merchant normal job only. From our data the dialogue shouldn't appear for high merchant.

Reply:
= is wrong anyway. it should be ==